app/controllers/skills_controller.rb

```diff
@@ -1,48 +0,0 @@
-class SkillsController < ApplicationController
-  def index
-    @skills = Skill.all
-  end
-
-  def new
-    @skill = Skill.new
-  end
-
-  def edit
-    @skill = Skill.find(params[:id])
-  end
-
-  def show
-    @skill = Skill.find(params[:id])
-  end
-
-  def create
-    @skill = Skill.new(skill_params)
```

Comment:
you dont required to use @skill unless you use it in the views
use just skill instead